### PR TITLE
925953 - WinUI Kanban control UG content

### PR DESCRIPTION
**Pull request by Kerubananthan3761**
## Description
 Added  WinUI Kanban control UG content.

### CODE REVIEW

File: winui/Kanban/Getting-started.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
+---
```
Comment (Karthickmani97) [inline winui/Kanban/Getting-started.md:1]: change file name to `getting-started`

File: winui/Kanban/Overview.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
```
Comment (Karthickmani97) [inline winui/Kanban/Overview.md:1]: rename file to `overview`

File: winui/Kanban/Overview.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+layout: post
+title: About WinUI Kanban control | Syncfusion
+description: Learn here all about introduction of Syncfusion Essential Studio WinUI Kanban (SfKanban) control, its features and more.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# WinUI Kanban (SfKanban) Overview
```
Comment (Karthickmani97) [inline winui/Kanban/Overview.md:10]: Overview of WinUI Kanban (SfKanban)

File: winui/Kanban/Overview.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+The Syncfusion WinUI Kanban control is a versatile and interactive tool designed to simplify task management and enhance workflow visualization. It offers clear and efficient way to monitor tasks across various stages of completion, ensuring seamless tracking of progress. With its elegant design and powerful features, the control facilitates effective planning and promotes productivity, making it an essential asset for managing tasks and workflows throughout the software development lifecycle.
```
Comment (Karthickmani97) [inline winui/Kanban/Overview.md:12]: The Syncfusion WinUI Kanban control is an efficient way to visualize a workflow at each stage of completion. It facilitates clear planning and provides a transparent view of work progression. SfKanban offers a variety of features that are essential for monitoring tasks throughout the software development cycle.

File: winui/Kanban/Overview.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+## Key features
+
+* Swimlanes – Horizontally organize the Kanban board to categorize cards by key attributes such as assignee, project, or type.
```
Comment (Karthickmani97) [inline winui/Kanban/Overview.md:16]: **Work-in-Progress (WIP) Limit**:- Allows users to set task limits for each stage, preventing overload and improving efficiency.
**Cards**: Display tasks, progress, and priority for better tracking.
**Columns**: Organize work into stages, such as To-Do, In-Progress, and Completed.
**Swimlanes**: Categorize tasks based on projects, teams, or custom criteria.
Interaction: Includes features like card tapping, column expansion/collapsing, and drag-and-drop for seamless workflow management.
**Events** : Triggered when dragging or repositioning Kanban cards, as well as when a card is tapped.
**Appearance Customization**: Supports UI customization for elements such as column headers, cards, and swimlane headers using templates.
**Localization and RTL support**: Easily adapts to any language or culture, with support for right-to-left (RTL) layout direction.
**Themes**: Offers built-in light and dark themes, which automatically adjust to the system's settings.

File: winui/Kanban/Getting-started.md [hunk 6]
```diff
@@ -0,0 +1,241 @@
...
+layout: post
+title: Getting Started with WinUI Kanban control | Syncfusion
+description: Learn here all about getting started with Syncfusion WinUI Kanban (SfKanban) control, its elements, and more.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+# Getting Started with WinUI Kanban (SfKanban)
```
Comment (Karthickmani97) [inline winui/Kanban/Getting-started.md:9]: Getting Started with WinUI Kanban Control

File: winui/Kanban/Getting-started.md [hunk 7]
```diff
@@ -0,0 +1,241 @@
...
+
+This section provides an overview of working with WinUI Kanban and a step-by-step guide to configuring and using it effectively in your application.
```
Comment (Karthickmani97) [inline winui/Kanban/Getting-started.md:11]: This section provides a quick overview of how to get started with the WinUI Kanban control (SfKanban) for WinUI and a walk-through to configure the WinUI Kanban control in a real-time scenario. Follow the steps below to add WinUI Kanban control to your project.

File: winui/Kanban/Getting-started.md [hunk 8]
```diff
@@ -0,0 +1,241 @@
...
+
+3. Import the control namespace `Syncfusion.UI.Xaml.Kanban` in XAML or C# code.
+
+4. Initialize the WinUI Kanban control.
+
+{% capture codesnippet1 %}
+{% tabs %}
+
+{% highlight xaml tabtitle="MainWindow.xaml" hl_lines="7 12" %}
+<Window x:Class="GettingStarted.MainWindow"
```
Comment (Karthickmani97) [inline winui/Kanban/Getting-started.md:27]: @Kerubananthan3761 , follow this standards

image

File: winui/Kanban/Getting-started.md [hunk 9]
```diff
@@ -0,0 +1,241 @@
...
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+{% endcapture %}
+{{ codesnippet1 | OrderList_Indent_Level_1 }}
+
+### Create a Data Model
```
Comment (Karthickmani97) [inline winui/Kanban/Getting-started.md:68]: Define the view model

File: winui/Kanban/Getting-started.md [hunk 10]
```diff
@@ -0,0 +1,241 @@
...
+
+To populate the `SfKanban` control, you need to create a collection of `KanbanModel` objects. These objects represent the tasks or items to be displayed on the Kanban board, each containing properties such as `title`, `description`, `category`, and other relevant details.
```
Comment (Karthickmani97) [inline winui/Kanban/Getting-started.md:70]: Create a ViewModel class with a collection property to hold multiple instances of the KanbanModel, where each KanbanModel instance represents a card in the Kanban control.

File: winui/Kanban/Getting-started.md [hunk 11]
```diff
@@ -0,0 +1,241 @@
...
+
+{% highlight c# %}
+   
+    /// <summary>
```
Comment (Karthickmani97) [inline winui/Kanban/Getting-started.md:74]: Remove summary

File: winui/Kanban/Getting-started.md [hunk 12]
```diff
@@ -0,0 +1,241 @@
...
+    /// Represents a ViewModel that designed to manage a collection of kanban tasks in a software project management system, typically used in kanban boards.
+    /// </summary>
+    public class GettingStartedViewModel
```
Comment (Karthickmani97) [inline winui/Kanban/Getting-started.md:77]: Rename class to `ViewModel`

File: winui/Kanban/Getting-started.md [hunk 13]
```diff
@@ -0,0 +1,241 @@
...
+            taskDetails.Add(taskDetail);
+            return taskDetails;
+        }
+
+        #endregion
+    }
+
+{% endhighlight %}
+
+### Bind Data to the Kanban Control
```
Comment (Karthickmani97) [inline winui/Kanban/Getting-started.md:174]: Bind item source for Kanban

File: winui/Kanban/Getting-started.md [hunk 14]
```diff
@@ -0,0 +1,241 @@
...
+            this.Content = sfKanban;
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+{% endcapture %}
+{{ codesnippet1 | OrderList_Indent_Level_1 }}
+
```
Comment (Karthickmani97) [inline winui/Kanban/Getting-started.md:67]: ## Populate WinUI kanban item source

Here are the steps to render kanban card items using the WinUI Kanban control with the respective `KanbanModel` class:

- Create the ViewModel.
- Bind the item source for Kanban.

File: winui/Kanban/Getting-started.md [hunk 15]
```diff
@@ -0,0 +1,241 @@
...
+        }
+
+        #endregion
+    }
+
+{% endhighlight %}
+
+### Bind Data to the Kanban Control
+
+To bind a data source to the `SfKanban` control, set the `ItemsSource` property as demonstrated in the example below. This property connects the collection of `KanbanModel` objects to the Kanban board, enabling the display of tasks.
```
Comment (Karthickmani97) [inline winui/Kanban/Getting-started.md:176]: To populate the Kanban card items, utilize the `ItemsSource` property of SfKanban.

File: winui/Kanban/Getting-started.md [hunk 16]
```diff
@@ -0,0 +1,241 @@
...
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<Grid>
+    <kanban:SfKanban x:Name="sfKanban"
```
Comment (Karthickmani97) [inline winui/Kanban/Getting-started.md:183]: Change `x:Name:kanban`

File: winui/Kanban/Getting-started.md [hunk 17]
```diff
@@ -0,0 +1,241 @@
...
+                     ItemsSource="{Binding TaskDetails}"/>
+    <Grid.DataContext>
```
Comment (Karthickmani97) [inline winui/Kanban/Getting-started.md:185]: Move DataContext code to under grid initialization

File: winui/Kanban/Getting-started.md [hunk 18]
```diff
@@ -0,0 +1,241 @@
...
+
+{% highlight c# %}
+
+this.sfKanban.ItemsSource = new GettingStartedViewModel().TaskDetails;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Define columns
```
Comment (Karthickmani97) [inline winui/Kanban/Getting-started.md:200]: Defining columns in the Kanban control

File: winui/Kanban/Getting-started.md [hunk 19]
```diff
@@ -0,0 +1,241 @@
...
+To configure columns in the `SfKanban` control, you must manually define them by adding `KanbanColumn` objects to the `Columns` collection property of SfKanban.
+
+The `ItemsSource` bound to the Kanban control will populate the respective columns based on the `Categories` property in KanbanColumn.
+
+The following code example illustrates how this can be done.
+
+{% tabs %}
+{% highlight xaml %}
+
+<kanban:SfKanban x:Name="sfKanban"
```
Comment (Karthickmani97) [inline winui/Kanban/Getting-started.md:211]: Change to `x:Name="kanban"`

File: winui/Kanban/Getting-started.md [hunk 20]
```diff
@@ -0,0 +1,241 @@
...
+
+Alternatively, you can set the `AutoGenerateColumns` property to `true` to automatically create columns without manually defining them. When enabled, the columns are generated dynamically based on the `Category` property of KanbanModel, creating a column for each distinct value in the `ItemsSource`.
+
+## Theme
+
+The WinUI Kanban supports light and dark themes, automatically adjusting to the system's theme settings for a consistent and visually appealing experience in any environment.
+
+Refer to the following link for guidance on applying themes: [Themes for Syncfusion WinUI controls](https://help.syncfusion.com/winui/common/themes)
+
+You can find the kanban keys for both themes here.
```
Comment (Karthickmani97) [inline winui/Kanban/Getting-started.md:241]: Add below 

**Note**

You can refer to our `WinUI Kanban` feature tour page for its groundbreaking feature representations. You can also explore our `WinUI Kanban Example` that shows you how to render the Kanban in WinUI.

**Review by Karthickmani97 — COMMENTED** [review]
@Kerubananthan3761 , address my review correction

File: winui/Kanban/Getting-started.md (hunk 1 above)
Comment (Kerubananthan3761) [reply winui/Kanban/Getting-started.md]: changed file name to getting-started

File: winui/Kanban/Overview.md (hunk 2 above)
Comment (Kerubananthan3761) [reply winui/Kanban/Overview.md]: renamed file to overview

File: winui/Kanban/Overview.md (hunk 3 above)
Comment (Kerubananthan3761) [reply winui/Kanban/Overview.md]: Changed to Overview of WinUI Kanban (SfKanban)

File: winui/Kanban/Overview.md (hunk 4 above)
Comment (Kerubananthan3761) [reply winui/Kanban/Overview.md]: Changed content

File: winui/Kanban/Overview.md (hunk 5 above)
Comment (Kerubananthan3761) [reply winui/Kanban/Overview.md]: changed content

File: winui/Kanban/Getting-started.md (hunk 6 above)
Comment (Kerubananthan3761) [reply winui/Kanban/Getting-started.md]: changed to Getting Started with WinUI Kanban Control

File: winui/Kanban/Getting-started.md (hunk 7 above)
Comment (Kerubananthan3761) [reply winui/Kanban/Getting-started.md]: changed content

File: winui/Kanban/Getting-started.md (hunk 8 above)
Comment (Kerubananthan3761) [reply winui/Kanban/Getting-started.md]: Example codes changed

File: winui/Kanban/Getting-started.md (hunk 9 above)
Comment (Kerubananthan3761) [reply winui/Kanban/Getting-started.md]: Changed to Define the view model

File: winui/Kanban/Getting-started.md (hunk 10 above)
Comment (Kerubananthan3761) [reply winui/Kanban/Getting-started.md]: content changed

File: winui/Kanban/Getting-started.md (hunk 11 above)
Comment (Kerubananthan3761) [reply winui/Kanban/Getting-started.md]: Summary removed

File: winui/Kanban/Getting-started.md (hunk 12 above)
Comment (Kerubananthan3761) [reply winui/Kanban/Getting-started.md]: Renamed class to ViewModel

File: winui/Kanban/Getting-started.md (hunk 13 above)
Comment (Kerubananthan3761) [reply winui/Kanban/Getting-started.md]: Changed to Bind item source for Kanban

File: winui/Kanban/Getting-started.md (hunk 14 above)
Comment (Kerubananthan3761) [reply winui/Kanban/Getting-started.md]: added content

File: winui/Kanban/Getting-started.md (hunk 15 above)
Comment (Kerubananthan3761) [reply winui/Kanban/Getting-started.md]: Content changed

File: winui/Kanban/Getting-started.md (hunk 17 above)
Comment (Kerubananthan3761) [reply winui/Kanban/Getting-started.md]: Moved DataContext code under grid initialization

File: winui/Kanban/Getting-started.md (hunk 18 above)
Comment (Kerubananthan3761) [reply winui/Kanban/Getting-started.md]: Changed to Defining columns in the Kanban control

File: winui/Kanban/Getting-started.md [hunk 21]
```diff
@@ -0,0 +1,241 @@
...
+
+this.sfKanban.ItemsSource = new GettingStartedViewModel().TaskDetails;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Define columns
+
+To configure columns in the `SfKanban` control, you must manually define them by adding `KanbanColumn` objects to the `Columns` collection property of SfKanban.
```
[the remark this answers is not in the document]
Comment (Kerubananthan3761) [reply winui/Kanban/Getting-started.md]: Content changed

File: winui/Kanban/Getting-started.md (hunk 19 above)
Comment (Kerubananthan3761) [reply winui/Kanban/Getting-started.md]: Changed to x:Name="kanban"

File: winui/Kanban/Getting-started.md (hunk 20 above)
Comment (Kerubananthan3761) [reply winui/Kanban/Getting-started.md]: Content changed.

File: winui/Kanban/overview.md [hunk 22]
```diff
@@ -0,0 +1,24 @@
...
+title: About WinUI Kanban control | Syncfusion
+description: Learn here all about introduction of Syncfusion Essential Studio WinUI Kanban (SfKanban) control, its features and more.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# Overview of WinUI Kanban (SfKanban)
+
+The Syncfusion WinUI Kanban control is an efficient way to visualize a workflow at each stage of completion. It facilitates clear planning and provides a transparent view of work progression. SfKanban offers a variety of features that are essential for monitoring tasks throughout the software development cycle.
```
Comment (Karthickmani97) [inline winui/Kanban/overview.md:12]: The Syncfusion WinUI Kanban control provides an efficient way to visualize workflows at each stage of completion. It enables clear planning and offers a transparent view of work progress. With a range of essential features, SfKanban is ideal for monitoring tasks throughout the software development cycle.

File: winui/Kanban/overview.md [hunk 23]
```diff
@@ -0,0 +1,24 @@
...
+
+## Key features
+
+* Work-in-Progress (WIP) Limit: Allows users to set task limits for each stage, preventing overload and improving efficiency.
```
Comment (Karthickmani97) [inline winui/Kanban/overview.md:16]: **Work-in-progress (WIP) limit**: Users can set task limits per stage, preventing overload and improving efficiency.
* **Cards**: Display tasks, progress, and priority.
* **Columns**: Organize work into columns for stages like to-do, in-progress, and completed.
* **Swim lanes**: Organize tasks based on projects, teams, or custom criteria.
* **Interaction**: Includes card tapping, column expansion/collapse, and drag-and-drop.
* **Events** : Triggered when cards are dragged, repositioned, or tapped.
* **Appearance customization**: Customize each element using templates, e.g., column headers, cards, and swim lane headers.
* **Localization and RTL**: Easily adapt the control to any supported language or culture. Render the board right-to-left (RTL) to suit RTL languages.
* **Themes**: Built-in light and dark themes automatically adjust to system settings.

File: winui/Kanban/getting-started.md [hunk 24]
```diff
@@ -0,0 +1,273 @@
...
+documentation: ug
+---
+
+# Getting Started with WinUI Kanban Control
+
+This section provides a quick overview of how to get started with the WinUI Kanban control (SfKanban) for WinUI and a walk-through to configure the WinUI Kanban control in a real-time scenario. Follow the steps below to add WinUI Kanban control to your project.
+
+## Creating an application with WinUI Kanban
+
+1. Create a [WinUI 3 desktop app for C# and .NET 5](https://learn.microsoft.com/en-us/windows/apps/winui/winui3/create-your-first-winui3-app).
```
Comment (Karthickmani97) [inline winui/Kanban/getting-started.md:16]: [WinUI 3 desktop app for C# and .NET 6](https://learn.microsoft.com/en-us/windows/apps/winui/winui3/create-your-first-winui3-app).

File: winui/Kanban/getting-started.md [hunk 25]
```diff
@@ -0,0 +1,273 @@
...
+4. Initialize the WinUI Kanban control.
+
+{% capture codesnippet1 %}
+{% tabs %}
+
+{% highlight XAML tabtitle="MainWindow.xaml" hl_lines="3 5" %}
+
+<Window 
+    ...
+    xmlns:kanban="using:Syncfusion.UI.Xaml.Kanban">
```
Comment (Karthickmani97) [inline winui/Kanban/getting-started.md:31]: Leave space, sentence should start with ... vertically in line num 30

File: winui/Kanban/getting-started.md [hunk 26]
```diff
@@ -0,0 +1,273 @@
...
+{% endhighlight %}
+{% endtabs %}
+{% endcapture %}
+{{ codesnippet1 | OrderList_Indent_Level_1 }}
+
+## Populate WinUI Kanban item source
+
+Here are the steps to render kanban card items using the WinUI Kanban control with the respective `KanbanModel` class:
+
+* Create the ViewModel.
```
Comment (Karthickmani97) [inline winui/Kanban/getting-started.md:62]: Create view model.

File: winui/Kanban/getting-started.md [hunk 27]
```diff
@@ -0,0 +1,273 @@
...
+* Bind the item source for Kanban.
+
+### Define the view model
```
Comment (Karthickmani97) [inline winui/Kanban/getting-started.md:65]: Create view model

File: winui/Kanban/getting-started.md [hunk 28]
```diff
@@ -0,0 +1,273 @@
...
+    }
+
+    #endregion
+}
+
+{% endhighlight %}
+
+### Bind item source for Kanban
+
+To populate the Kanban card items, utilize the `ItemsSource` property of SfKanban.
```
Comment (Karthickmani97) [inline winui/Kanban/getting-started.md:184]: To populate the Kanban card items, utilize the `ItemsSource` property of `SfKanban`.

File: winui/Kanban/getting-started.md [hunk 29]
```diff
@@ -0,0 +1,273 @@
...
+2. Add reference to the `Syncfusion.Kanban.WinUI` NuGet.
+
+3. Import the control namespace `Syncfusion.UI.Xaml.Kanban` in XAML or C# code.
+
+4. Initialize the WinUI Kanban control.
+
+{% capture codesnippet1 %}
+{% tabs %}
+
+{% highlight XAML tabtitle="MainWindow.xaml" hl_lines="3 5" %}
```
Comment (Karthickmani97) [inline winui/Kanban/getting-started.md:27]: Set title as XAML and C#

image

File: winui/Kanban/getting-started.md [hunk 30]
```diff
@@ -0,0 +1,273 @@
...
+
+{% highlight C# tabtitle="MainWindow.xaml.cs" hl_lines="2" %}
+
+this.kanban.ItemsSource = new ViewModel().TaskDetails;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Automatically generating columns in the Kanban control
```
Comment (Karthickmani97) [inline winui/Kanban/getting-started.md:207]: ## Defining columns

File: winui/Kanban/getting-started.md [hunk 31]
```diff
@@ -0,0 +1,273 @@
...
+
+To automatically generate columns in the Kanban control, you can set the `AutoGenerateColumns` property to `true`. When enabled, the columns are generated dynamically based on the `Category` property of KanbanModel, creating a column for each distinct value in the `ItemsSource`.
```
Comment (Karthickmani97) [inline winui/Kanban/getting-started.md:209]: The columns are generated automatically based on the different values of the `Category` in the `KanbanModel` class from the `ItemsSource`. However, you can manually define the columns by setting the `AutoGenerateColumns` property to `false` and adding `KanbanColumn` instances to the `Columns` property of `SfKanban`. You can define the column categories using the `Categories` property of `KanbanColumn`, and the cards will be added to their respective columns.

File: winui/Kanban/getting-started.md [hunk 32]
```diff
@@ -0,0 +1,273 @@
...
+
+{% highlight C# tabtitle="MainWindow.xaml.cs" hl_lines="2" %}
+
+this.kanban.ItemsSource = new ViewModel().TaskDetails;
+this.kanban.AutoGenerateColumns = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+### Manually defining columns in the Kanban control
```
Comment (Karthickmani97) [inline winui/Kanban/getting-started.md:232]: ## Defining columns

File: winui/Kanban/getting-started.md [hunk 33]
```diff
@@ -0,0 +1,273 @@
...
+
+You can manually define columns in the Kanban control by adding `KanbanColumn` objects to the `Columns` collection property of SfKanban. The `ItemsSource` bound to the Kanban control will populate the respective columns based on the `Categories` property in `KanbanColumn`.
```
Comment (Karthickmani97) [inline winui/Kanban/getting-started.md:234]: The columns are generated automatically based on the different values of the `Category` in the `KanbanModel` class from the `ItemsSource`. However, you can manually define the columns by setting the `AutoGenerateColumns` property to `false` and adding `KanbanColumn` instances to the `Columns` property of `SfKanban`. You can define the column categories using the `Categories` property of `KanbanColumn`, and the cards will be added to their respective columns.

File: winui/Kanban/getting-started.md [hunk 34]
```diff
@@ -0,0 +1,273 @@
...
+
+The following code example illustrates how this can be done.
```
Comment (Karthickmani97) [inline winui/Kanban/getting-started.md:236]: Remove this

File: winui/Kanban/getting-started.md [hunk 35]
```diff
@@ -0,0 +1,273 @@
...
+
+{% tabs %}
+{% highlight XAML tabtitle="MainWindow.xaml" hl_lines="3 4 5 6" %}
+
+<kanban:SfKanban x:Name="kanban"
+                 ItemsSource="{Binding TaskDetails}"
+                 AutoGenerateColumns="False">
+    <kanban:KanbanColumn Categories="Open" HeaderText="To Do"/>
```
Comment (Karthickmani97) [inline winui/Kanban/getting-started.md:244]: Update code

` <kanban:SfKanban x:Name="kanban"
                  AutoGenerateColumns="False"
                  ItemsSource="{Binding TaskDetails}">
     <kanban:KanbanColumn HeaderText="To Do" Categories="Open" />
     <kanban:KanbanColumn HeaderText="In Progress" Categories="In Progress" />
     <kanban:KanbanColumn HeaderText="Code Review" Categories="Code Review" />
     <kanban:KanbanColumn HeaderText="Done" Categories="Done" />
 </kanban:SfKanban>`

File: winui/Kanban/getting-started.md [hunk 36]
```diff
@@ -0,0 +1,273 @@
...
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+{% highlight C# tabtitle="MainWindow.xaml.cs" hl_lines=" 2 4 5 6" %}
+
+this.kanban.ItemsSource = new ViewModel().TaskDetails;
+this.kanban.AutoGenerateColumns = false;
+
+this.kanban.Columns.Add(new KanbanColumn() { Categories = "Open", HeaderText = "To Do", });
```
Comment (Karthickmani97) [inline winui/Kanban/getting-started.md:256]: Modify code as i suggested in XAML

File: winui/Kanban/getting-started.md [hunk 37]
```diff
@@ -0,0 +1,273 @@
...
+Here are the steps to render kanban card items using the WinUI Kanban control with the respective `KanbanModel` class:
+
+* Create the ViewModel.
+* Bind the item source for Kanban.
+
+### Define the view model
+
+Create a view model class to set values for the properties listed in the `KanbanModel` class as shown in the following example code. Each `KanbanModel` instance represents a card in the Kanban control.
+
+{% highlight c# %}
```
Comment (Karthickmani97) [inline winui/Kanban/getting-started.md:69]: Set tab title to ViewModel

image

File: winui/Kanban/getting-started.md [hunk 38]
```diff
@@ -0,0 +1,273 @@
...
+
+## Theme
+
+The WinUI Kanban supports light and dark themes, automatically adjusting to the system's theme settings for a consistent and visually appealing experience in any environment.
+
+Refer to the following link for guidance on applying themes: [Themes for Syncfusion WinUI controls](https://help.syncfusion.com/winui/common/themes)
+
+N> You can refer to our `WinUI Kanban` feature tour page for its groundbreaking feature representations. You can also explore our `WinUI Kanban Examples` that shows you how to render the Kanban in WinUI.
+
+Also, you can find the Kanban keys for the Light, Dark, and High Contrast themes here.
```
Comment (Karthickmani97) [inline winui/Kanban/getting-started.md:273]: Remove this line, add our resource keys in https://github.com/syncfusion/winui-controls-theme-resource-files

**Pull request merged**